swift/StableDiffusion/pipeline/StableDiffusionPipeline+Resources.swift

```diff
@@ -29,6 +33,10 @@ public extension StableDiffusionPipeline {
             safetyCheckerURL = baseURL.appending(path: "SafetyChecker.mlmodelc")
             vocabURL = baseURL.appending(path: "vocab.json")
             mergesURL = baseURL.appending(path: "merges.txt")
+            controlNetDirURL = baseURL.appending(path: "Controlnet")
```

Comment:
Since torch2coreml seems to export to the `controlnet` directory, it seems like a good idea to start with lower case here as well.

> Thanks for your great contribution!